src/core_plugins/metrics/public/components/vis_editor_visualization.js

```diff
@@ -46,11 +45,6 @@ class VisEditorVisualization extends Component {
     window.removeEventListener('mouseup', this.handleMouseUp);
   }
 
-  componentDidMount() {
-    const el = findDOMNode(this.visDiv);
```

Comment:
By removing this, you don't need the `visDiv` reference anymore, so you can remove setting it from the `div` below.